File: beecrowd/3301.py
a,b,c = input().split()
a=int(a)
b=int(b)
c=int(c)
if b>a and b<c:
    print("zezinho")
elif b<a and b>c:
    print("zezinho")
elif c<a and c>b:
    print("luisinho")
elif c>a and c<b:
    print("luisinho")
else:
    print("huguinho")

File: beecrowd/3301.cpp
#include<iostream>
using namespace std;
int main()
{
    int a,b,c;
    cin>>a>>b>>c;
    if (b>a && b<c)
    cout<<"zezinho"<<endl;
else if( b<a &&  b>c)
    cout<<"zezinho"<<endl;
else if( c<a &&  c>b)
    cout<<"luisinho"<<endl;
else if (c>a && c<b)
    cout<<"luisinho"<<endl;
else
    cout<<"huguinho"<<endl;
}